httpd/handler.go

```diff
@@ -127,16 +127,6 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *influ
 		return
 	}
 
-	// If authentication is enabled and there are no users yet, make sure
-	// the first statement is creating a new cluster admin.
-	if h.requireAuthentication && h.server.UserCount() == 0 {
```

Comment:
This is no longer required now that we have only two modes, either authless, or authentication enabled.